src/metadata.cc

```diff
@@ -0,0 +1,33 @@
+#include "metadata.hh"
+#include "toml.hpp"
+#include <QtDebug>
```

Comment:
what is this?
`QDebug`?

Reply:
They are the same, both just include "qtdebug.hh" 

Why it exist :sweat_smile: 

Changed to QDebug